pkg/operator/cluster/cluster.go

```diff
@@ -133,7 +130,8 @@ func (c *Cluster) createClientAccess(clusterInfo *cephmon.ClusterInfo) error {
 	defer conn.Shutdown()
 
 	// create a user for rbd clients
-	username := "client.rook-rbd-user"
+	name := fmt.Sprintf("%s-rbd-user", c.Namespace)
```

Comment:
Should we change the user to namespace-rook-user instead? I see rbd as a detail in rook that we want to hide from the user